<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(18, 18, 18);
	background-image: url('./img/background.png');
}

QMenuBar {
    background-color: qlineargradient(spread:pad, x1:0.472, y1:1, x2:0.482955, y2:0, stop:0 rgba(18, 18, 18, 123), stop:0.795455 rgba(35, 35, 35, 255));
    spacing: 3px;
}

QMenuBar::item {
	color: #FFFFFF;
    padding: 1px 4px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected {
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}

QMenu::separator {
    height: 2px;
    background: #353535;
}

QMenu {
	color: #FFFFFF;
	background-color: #121212;
}

QMenu::item:selected {
	color: #FFFFFF;
	background-color: #303030;
}

QWidget {
	border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="buttonsFrame">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #121212;

	border: solid;
	border-color: rgb(45,55,50);
	border-width: 2px;
}

QPushButton {
	color: white;

	width: 300px;
	height: 20px;
	
	border: solid;
	border-color: #353535;
	border-width: 1px;
	border-radius: 8px;
}

QPushButton:hover {
	border-color: rgb(170, 170, 170);
	border-width: 2px;

	width: 300px;
	height: 30px;
}

QPushButton:pressed {
	border-color: white;
	border-width: 2px;
	
	width: 300px;
	height: 30px;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="buttons">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>75</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>75</number>
       </property>
       <item>
        <widget class="QPushButton" name="calcBtn">
         <property name="text">
          <string>Калькулятор</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">tabsButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="graphBtn">
         <property name="text">
          <string>Графики</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">tabsButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eqBtn">
         <property name="text">
          <string>Уравнения</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">tabsButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forLibBtn">
         <property name="text">
          <string>Библиотека формул</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">tabsButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bookLibButton">
         <property name="text">
          <string>Библиотека учебников</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">tabsButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="textFrame">
      <property name="minimumSize">
       <size>
        <width>490</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="mainLabel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	padding: 5px 5px 5px 5px;
	color: white;
	border: solid;
	border-color: #CCCCCC;
	border-width: 2px;
	border-radius: 16px;
}</string>
         </property>
         <property name="text">
          <string>Universal Helper</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Open Sans</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {color: white}</string>
         </property>
         <property name="text">
          <string>Выберите вкладку слева</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="widgetsLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="closeWinsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="authorsAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="closeWinsAction">
   <property name="text">
    <string>Закрыть все окна</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="authorsAction">
   <property name="text">
    <string>Авторы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tabsButtonGroup"/>
 </buttongroups>
</ui>
